apps/explorer/config/test.exs

```diff
@@ -7,6 +7,7 @@ config :bcrypt_elixir, log_rounds: 4
 config :explorer, Explorer.Repo,
   database: "explorer_test",
   hostname: "localhost",
+  password: "1234",
```

Comment:
just for tests, right?

Reply:
Yes, just for tests. And in addition is overridden in some tests with an env variable (as in circleci).